packages/frontapp/src/app/components/Modal/index.tsx

```diff
@@ -54,10 +55,14 @@ export const ContentModal = styled.div<IStyleProps>`
   width: 100%;
   margin: auto;
   @media (max-width: ${dimensions.phone_width}) {
-    max-height: 100vh;
+    max-height: 100%;
     padding: ${dimensions.xl} ${dimensions.xs_2};
     border-radius: ${dimensions.base_2} ${dimensions.base_2} 0 0;
     margin: 0;
+    overflow-y: scroll;
```

Comment:
Why scroll and not auto?
Scroll always shows scrollbar, auto - only when needed

Reply:
Nope, that's not how it works. `Overflow: auto` we can't use here as it set the fixed value to our modal, so it can not be scrolled down if it has too much content within it (only Mozilla provide the scroll when needed), so here I added scrollable modal and hided the scrollbar